pkg/auth/auth.go

```diff
@@ -176,12 +177,18 @@ func (u *UserAuth) isAllowed(verb string, itemsToCheck []resource) ([]Action, er
 			}
 		}
 		if !allowed {
-			rejectedActions = append(rejectedActions, Action{
+			rejectedAction := Action{
 				APIVersion: i.APIVersion,
 				Resource:   resource,
-				Namespace:  i.Namespace,
 				Verbs:      []string{verb},
-			})
+			}
+			if namespaced {
+				rejectedAction.ClusterWide = false
```

Comment:
I believe this is not necessary because of the bool default value. I am not opposed to be explicit in any case.